lib/resty/core/output.lua

```diff
@@ -0,0 +1,50 @@
+local ffi = require "ffi"
+local base = require "resty.core.base"
+local FFI_OK = base.FFI_OK
+local ffi_cast = ffi.cast
+local const_char_ptr = ffi.typeof("const char *")
+local C = ffi.C
+--local get_string_buf = base.get_string_buf
+--local get_size_ptr = base.get_size_ptr
+
+local output = {}
```

Comment:
Also, we need a version number from `resty.core.base` for this module. See other modules for examples.